x-pack/plugins/canvas/canvas_plugin_src/renderers/advanced_filter/component/advanced_filter.tsx

```diff
@@ -6,7 +6,7 @@
 
 import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
 import PropTypes from 'prop-types';
-import React, { SFC } from 'react';
+import React, { FunctionComponent } from 'react';
```

Comment:
I keep seeing this in your PR's... mind sharing the reason for using one type instead of another?

Reply:
`SFC` was deprecated, and we couldn't use `FunctionComponent` from `@types/react` until our React dependency was updated to at least 16.7.

Reply:
You can also refer to [this Twitter thread](https://twitter.com/dan_abramov/status/1057625147216220162) from Abramov, and this particular reply:

![Screen Shot 2019-04-03 at 7 01 23 PM](https://user-images.githubusercontent.com/297604/55520683-0351c980-5643-11e9-8dba-9226cdf257f4.png)

Reply:
Associated type pull for anyone else that comes across this: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/30364